tools/cloud-build/daily-tests/tests/slurm-v5.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
```

Comment:
Nit, but you should really call this slurm-v5-centos7, since you will need another one of these to add ubuntu as well. Won't you?